e2e/cypress/integration/scroll/fixed_width_spec.js

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const timeouts = require('../../fixtures/timeouts');
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @scroll
```

Comment:
Suggesting an extra line after the `// Group` line:
```suggestion
// Group: @scroll

```

Reply:
:heavy_check_mark: